packages/trader/src/Modules/CFD/Components/cfd-account-card.jsx

```diff
@@ -59,14 +59,26 @@ const PasswordBox = ({ platform, onClick }) => (
                     }
                 )}
                 classNameBubble='cfd-account-card__password-tooltip'
-                margin={35}
             >
                 <Text size='xs'>•••••••••••••••</Text>
             </Popover>
         </div>
-        <Button className='cfd-account-card__password-action' tertiary onClick={onClick}>
-            {localize('Change')}
-        </Button>
+        <Popover alignment='bottom' message={localize('Change Password')}>
+            <Button
+                className='cfd-account-card__password-action toolbar__btn--icon'
+                transparent
+                onClick={onClick}
+                icon={
+                    <Icon
+                        icon='IcEdit'
+                        className='da-article__learn-more-icon'
+                        height={16}
+                        width={16}
```

Comment:
Is it necessary to set `height` and `width` values? if I'm not mistaken, the default size is `16`

Reply:
you're right, removed